osu.Framework/Game.cs

```diff
@@ -265,7 +265,7 @@ protected override void LoadComplete()
             FrameStatistics.BindValueChanged(e => performanceOverlay.State = e.NewValue, true);
         }
 
-        protected readonly Bindable<FrameStatisticsMode> FrameStatistics = new Bindable<FrameStatisticsMode>();
+        protected internal readonly Bindable<FrameStatisticsMode> FrameStatistics = new Bindable<FrameStatisticsMode>();
```

Comment:
What's this for? None of the new tests seem to be using this?

```suggestion
        protected readonly Bindable<FrameStatisticsMode> FrameStatistics = new Bindable<FrameStatisticsMode>();
```

Reply:
Drive-by change, whoops.